doc/lua_api.txt

```diff
@@ -67,6 +67,10 @@ The game directory can contain the following files:
     * `disallowed_mapgens = <comma-separated mapgens>`
       e.g. `disallowed_mapgens = v5,v6,flat`
       These mapgens are removed from the list of mapgens for the game.
+    * `disallowed_settings = <comma-separated mapgens>`
+      e.g. `disallowed_settings = mgv5_spflags`
```

Comment:
Q: Could the game's `minetest.conf` set the value of `mgv5_spflags` to - say `nocaverns` - so that it is used in the generated world?